source/grid/tria.cc

```diff
@@ -12007,8 +11994,6 @@ Triangulation<dim, spacedim>::clear_despite_subscriptions()
   // This is the former function
   // clear without the assertion in
   // the beginning.
-  for (unsigned int i=0; i<levels.size(); ++i)
```

Comment:
I looked back over the other functions for clearing out a triangulation and I don't see any asserts left. git-blame tells me that this comment was last modified 5 years ago. Am I right in guessing it should just be removed?